### DISCUSSION TITLE
How to change font size of the secondary side bar

### DISCUSSION THREAD

**Discussion by addaboi** [posit-dev/positron#4312] [Q&A]
Using ctrl+ to zoom in the entire interface to a level where I can comfortably read the text in the secondary side bar increases the sizes of other elements of the window to a point where I lose too much screen space, so this is not a solution.

I can find settings to adjust the font in the editor but nothing for anything in the secondary sidepanel. Is there there an extension i can use or a css file or something I can change to just increase the font size there?

**Comment by seeM** [comment]
Thanks for the feedback @addaboi! I don't think it's possible to control the font size of the secondary sidebar at the moment. I've created issue https://github.com/posit-dev/positron/issues/4322 to track this work.
